site/socket/index.php

```diff
@@ -8,7 +8,7 @@
 use app\libraries\Core;
 use app\libraries\socket\Server;
 
-require_once(__DIR__.'/../vendor/autoload.php');
+require_once(__DIR__ . '/../vendor/autoload.php');
```

Comment:
This failed the PHP linter when I ran it locally, which is a bit weird since it looked like it passed when I checked when it was added and my vagrant instance was built from a relatively freshly downloaded repo

Reply:
Yeah looks like its fixed on master, updating the branch should fix this